docs/core/porting/unsupported-apis.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Unsupported APIs on .NET Core
+description: Learn which APIs from the .NET Framework that always throw an exception on .NET Core.
+ms.date: 12/23/2019
+---
+# APIs that always throw exceptions on .NET Core
+
+The following APIs will always through a <xref:System.PlatformNotSupportedException> when run on .NET Core.
```

Comment:
I know you’re still working on this but we need to a note that this list might not be complete. The report I got was an old one.